src/app/components/Spinner.jsx

```diff
@@ -0,0 +1,260 @@
+import Image from "next/image";
+
+export default function Spinner({
+  shiftRight,
+  shiftLeft,
+  dishes,
+  wheel,
+  price,
+  name,
+  description,
+  orderNow,
+  container,
+}) {
+  let contentSelector = 0;
+  let isOrange = true;
+  let newX = 0.0;
+  let newY = 0.0;
+  let spin = 0.0;
+
+  dishes.forEach((dish, i) => {
+    const angle = (2 * Math.PI * i) / dishes.length;
+    const spinRadius = 250.0;
+
+    newX = Math.cos(angle) * spinRadius;
+    newY = -1.0 * Math.sin(angle) * spinRadius;
+
+    dish.style.left = `50%`;
+    dish.style.top = `50%`;
+    dish.style.transform = `translate(-50%, -50%) translate(${newX}px, ${newY}px)`;
+
+    addDishesBackground(dish, i);
+  });
+
+  function addDishesBackground(dish, i) {
+    const images = [
+      "url('/img/e9df498957f2d3c23586a9edad79c926-fotor-2024020713573.png')",
+      "url('/img/dish_2.webp')",
+      "url('/img/dish_3.webp')",
+    ];
+    switch (i) {
+      case 0:
+      case 3:
+      case 6:
+        dish.style.backgroundImage = images[1];
+        break;
+      case 1:
+      case 4:
+      case 7:
+        dish.style.backgroundImage = images[2];
+        break;
+      case 2:
+      case 5:
+      case 8:
+        dish.style.backgroundImage = images[0];
+        break;
+    }
+  }
+
+  const shiftRightBtn = (e) => {
+    contentSelector++;
+    if (contentSelector == dishes.length) contentSelector = 0;
+
+    spinRight();
+    changeContent(contentSelector);
+    animateContent();
+    changeColors();
+    console.log(contentSelector);
+  };
+
+  const shiftLeftBtn = () => {
+    if (contentSelector != 0) contentSelector--;
+    else {
+      contentSelector = dishes.length - 1;
+    }
+    spinLeft();
+    animateContent();
+    changeContent(contentSelector);
+    changeColors();
+  };
+
+  function spinRight() {
+    let slice = 45;
+    spin += slice;
+    wheel.style.transform = `translate(-50%, -50%) rotate(${spin}deg)`;
+  }
+
+  function spinLeft() {
+    let slice = 45;
+    spin -= slice;
+    wheel.style.transform = `translate(-50%, -50%) rotate(${spin}deg)`;
+  }
```

Comment:
These two are almost the same, so why not create one instead and send the `direction` parameter?
```suggestion
  function spin(direction) {
    const slice = 45 * (direction === 'left' ? -1 : 1);
    spin += slice;
    wheel.style.transform = `translate(-50%, -50%) rotate(${spin}deg)`;
  }
```

And you use it like this: `spin('left')` or `spin('right')`